### PR TITLE
Add new operators to test

### PR DESCRIPTION
**Pull request by rajansandeep**
Adds the newly added flannel, kube-proxy and nodelocaldns operators to test script.

### CODE REVIEW

**Comment by justinsb** [comment]
Thanks @rajansandeep ... though it also looks like these tests aren't passing yet...

/approve
/lgtm

File: hack/unit-test.sh [hunk 1]
```diff
@@ -29,5 +29,14 @@ make test -C coredns
 echo "****** Testing Dashboard Operator ******"
 make test -C dashboard
 
+echo "****** Testing Flannel Operator ******"
+make test -C flannel
+
+echo "****** Testing Kube-Proxy Operator ******"
+make test -C kubeproxy
```
Comment (justinsb) [inline hack/unit-test.sh:36]: So looking at the failure, I think it's picking up the KUBERNETES_SERVICE_HOST from prow!

I think we can just unset KUBERNETES_SERVICE_HOST before calling this e.g. `KUBERNETES_SERVICE_HOST= make test -C kubeproxy`

(Or just `unset KUBERNETES_SERVICE_HOST` at the top of hack/unit-test.sh)

Comment (rajansandeep) [reply hack/unit-test.sh:36]: Aah yes, I'll add that!

**Comment by justinsb** [comment]
Thanks & sorry for the delay in merging this one!

/approve
/lgtm

**Pull request merged**